pkg/validation/spec/info.go

```diff
@@ -87,6 +88,31 @@ func (e Extensions) GetObject(key string, out interface{}) error {
 	return nil
 }
 
+func (e Extensions) sanitize() {
```

Comment:
does this mean all keys/values get added to this map and then non-extension keys get truncated? is there a way to avoid adding keys that are explicitly in peer struct types to this map?

Reply:
Yes, we end up truncating non-extension keys. I did not see a way in the go-json-experiment library to only include peer keys? Maybe @dsnet can advise?

Reply:
ok... it seems like this is an opportunity for follow-up improvements, either by not decoding as many things into the extensions map, or by making the decoding cheaper prior to filtering to extension keys (e.g. decoding to a `map[string]json.RawMessage` → filtering to extension keys → actually decoding the extension values to `interface{}`)

Reply:
probably worth making a note or filing an issue so we don't forget